tools/automation/twiml-generator/generator.py

```diff
@@ -24,12 +26,82 @@ def generate_code_sample_filepath(twiml_filepath, language):
     return twiml_filepath.parent.parent / (twiml_filepath.name[:-3] + version + language_spec['extension'])
 
 
+def test(code_generator):
```

Comment:
This makes me feel we are duplicating efforts and logic to run tests on the Twiml generated code. The only difference from this and the current snippet-tester is the time in which it runs (sooner or later) and the fact that this one does not report anything on CI.

I believe having a report on CI (and locally is we configure the environment) is more than enough to gain a feedback of things going wrong with the generation. WDYT?